test/common/json/json_fuzz_test.cc

```diff
@@ -0,0 +1,67 @@
+#include "common/common/assert.h"
+#include "common/json/json_loader.h"
+#include "common/protobuf/protobuf.h"
+#include "common/protobuf/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+// Strong assertion that applies across all compilation modes and doesn't rely
+// on gtest, which only provides soft fails that don't trip oss-fuzz failures.
+#define FUZZ_ASSERT(x) RELEASE_ASSERT(x, "")
```

Comment:
I see this used in other PRs also. Should we have a common fuzz helper utility include?

Reply:
Good idea, done.